python/flask/unsanitized_input.yaml

```diff
@@ -0,0 +1,39 @@
+rules:
+  - id: response_contains_unsanitized_input
+    patterns:
+      - pattern-either:
```

Comment:
I would probably add `pattern-inside` so that you are only catching on function scope

Reply:
Unfortunately this doesn't work, because metavariables aren't the same between different patterns :(